pkgs/applications/audio/yasr/40_dectalk_extended_chars

```diff
@@ -0,0 +1,14 @@
+Index: yasr-0.6.9/yasr/tts.c
+===================================================================
+--- yasr-0.6.9.orig/yasr/tts.c	2008-02-03 00:10:07.000000000 +1100
++++ yasr-0.6.9/yasr/tts.c	2012-02-05 10:59:06.059007839 +1100
+@@ -281,6 +281,9 @@
+   char *p = synth[tts.synth].unspeakable;
+ 
+   if (ch < 32) return 1;
++  /* characters with high bit set cause DECTALK to crash */
++  if (tts.synth == TTS_DECTALK && ch & 0x80)
++    return 1;
```

Comment:
As I made in a comment, this piece is not written by me. I blindly took it from the YASR debian source package. I just checked that this fix is implemented in the current HEAD version (2013) of YASR [https://sourceforge.net/p/yasr/git/ci/master/tree/yasr/tts.c#l286)]. So I guess the author is well aware of all these patches, and it is just that the official release is stuck at the old version v0.6.9 in 2008.